Author: misaka00251 <misaka00251@misakanet.cn>
Date:   Mon Aug 29 11:57:44 2022 +0800

	Fix support for RISC-V.

diff --git a/3rdparty/enca-1.19/config.guess b/3rdparty/enca-1.19/config.guess
index 1659250..d1bce03 100755
--- a/3rdparty/enca-1.19/config.guess
+++ b/3rdparty/enca-1.19/config.guess
@@ -1022,6 +1022,9 @@ EOF
     ppcle:Linux:*:*)
 	echo powerpcle-unknown-linux-${LIBC}
 	exit ;;
+    riscv32:Linux:*:* | riscv32be:Linux:*:* | riscv64:Linux:*:* | riscv64be:Linux:*:*)
+	echo $UNAME_MACHINE-unknown-linux-${LIBC}
+	exit ;;
     s390:Linux:*:* | s390x:Linux:*:*)
 	echo ${UNAME_MACHINE}-ibm-linux-${LIBC}
 	exit ;;
